pkgs/build-support/setup-hooks/audit-blas.sh

```diff
@@ -0,0 +1,35 @@
+# Ensure that we are always linking against “libblas.so.3” and
+# “liblapack.so.3”.
+
+auditBlas() {
+    local dir="$prefix"
+    [ -e "$dir" ] || return 0
+
+    local i
+    while IFS= read -r -d $'\0' i; do
+        if ! isELF "$i"; then continue; fi
+
+        if $OBJDUMP -p "$i" | grep 'NEEDED' | awk '{ print $2; }' | grep -q '\(libmkl_rt.so\|libopenblas.so.0\)'; then
+            echo "Found direct BLAS/LAPACK implementation reference in $i. Depend on blas / lapack instead."
```

Comment:
This error should be a bit more actionable. It should explain *why* you should depend on `blas` / `lapack` instead and how you can still use a specific implementation if you know what you're doing.